backend/configs/usr/lib/cgi-bin/download_rootfs.sh

```
@@ -0,0 +1,41 @@
+
+if [ -z "$(ls -A $MOUNT_DIR)" ]; then
+    echo "Status: 500"
+    echo ""
+    echo "Error mounting rootfs"
+else
+    echo "Status: 200"
+    echo "Content-Disposition: attachment; filename=\"rootfs_${SERIAL}_${date}.tar.gz\""
+    echo "Content-Type: application/octet-stream"
+    echo ""
+    cd "$MOUNT_DIR"
+    sudo tar --exclude='tmp' --exclude='var/tmp' -cf - * | run_pigz
```

Shellcheck (0.10.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)